### DISCUSSION TITLE
Is there a way to avoid tags to appear in the navigation menu?

### DISCUSSION THREAD

**Discussion by gciatto** [thegeeklab/hugo-geekblog#67] [Q&A]

**Comment by xoxys** [comment]
Right now, that's not possible. Can you describe your use-case? Why would you add a tag if the user cant use it for navigation?

**Reply by gciatto** [reply]
Currently, it is possible to add custom pages in the navigation bar (by editing the `data/menu/extra.yml`) but, apparently:
- tags appear in alphabetic order
- there is no way to avoid some tags are shown
- extra buttons always appear "after" the buttons automatically generated from tags

I would like a little bit more control on the navigation bar.

I'm imaging a navigation bar for a site describing a research, where the first buttons are fixed, eg:
1. Home
2. About
3. People
4. Promo video
5. Papers
6. Deliverables

Pages 1-4 are "fixed" and I would like the full pages to appear as soon as the corresponding buttons in the navigation bar are clicked. Pages 5-6 are "default" containers of pages tagged as either "paper" or "deliverable". I would also like menu entries to appear in that particular order.
There may be further tags for which the corresponding list of pages must not be visible in the navigation menu, as those lists are made accessible otherwise.
For instance, I would like to tag papers by year or delierables by organization involved, yet I don't want the navigation bar to explode.

**Reply by xoxys** [reply]
Got it, thanks for your explanation.
